android/src/main/java/com/vonovak/AddCalendarEventModule.java

```diff
@@ -104,6 +104,12 @@ private void presentEventAddingActivity(ReadableMap config) {
                     && config.getString("description") != null) {
                 calendarIntent.putExtra("description", config.getString("description"));
             }
+
+            if (config.hasKey("allDay")
+                    && config.getBoolean("allDay")) {
```

Comment:
Hello,
I modified the code with ` if (config.hasKey("allDay") && config.getType("allDay") == ReadableType.Boolean) {` in order to avoid app crash if value is not a Boolean

Reply:
I think it is actually good for the app to crash if we provide the "allDay" key and a value other than true of false